conda.bzl

```diff
@@ -101,22 +109,32 @@ def _create_conda_build_file(rctx, executable):
 
 def _load_conda_impl(rctx):
     installer = _download_conda(rctx)
-    executable = _install_conda(rctx, installer)
-    _update_conda(rctx, executable)
-    _create_conda_build_file(rctx, executable)
+    conda_executable = _install_conda(rctx, installer)
+    if rctx.attr.install_mamba:
+        _install_mamba(rctx, conda_executable)
+    _update_conda(rctx, conda_executable)
```

Comment:
Let's first update `conda` and then install `mamba`. `mamba` depends on `conda` so it's better to fully setup `conda` first